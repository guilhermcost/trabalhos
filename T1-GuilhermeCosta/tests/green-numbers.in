0 1 2 3 4 5 6 7 8 9 
13579
24680
924a   // retorna 924 e a
