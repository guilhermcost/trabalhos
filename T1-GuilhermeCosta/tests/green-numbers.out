(1,NUM,"0")
(1,NUM,"1")
(1,NUM,"2")
(1,NUM,"3")
(1,NUM,"4")
(1,NUM,"5")
(1,NUM,"6")
(1,NUM,"7")
(1,NUM,"8")
(1,NUM,"9")
(2,NUM,"13579")
(3,NUM,"24680")
(4,NUM,"924")
(4,ID,"a")
